### DISCUSSION TITLE
STATUS_ACCESS_VIOLATION is thrown if two program(same) run at the same time

### DISCUSSION THREAD

**Discussion by gaoe6661** [cefsharp/CefSharp#4855] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

126.2.70

### Operating System

Windows 11

### Architecture

x86

### .Net Version

6.0

### Implementation

WPF

### Reproduction Steps

* create a program that uses CEFSharp
* the cefsharp will load a local resource at `html\main.html`
* compile (i just tested under debug mode)
* copy one from `C:\---\bin\Debug\net6.0-windows` to `C:\---\bin\net6.0-windows`
* run one
* run another
* It will show the error

![image](https://github.com/cefsharp/CefSharp/assets/124422444/b99fe288-71e7-495d-b2b0-6c8369bb5447)

### Expected behavior

no error

### Actual behavior

have error

### Regression?

idk

### Does this problem also occur in the CEF Sample Application

Not Tested

### Other information

idk how to test by using it

**Comment by amaitland** [comment]
The chromium process Singleton only allows a single process to access a cache. If Cef.Initialize returns false then you cannot use the browser. Use a different cache path per process or implement the process Singleton 

See https://github.com/cefsharp/CefSharp/issues/4668 for more details
